### PR TITLE
[FEATURE] Rest API upload file returns message object

### LINKED ISSUE #13808: Rest API upload file get message
Hello,

I’m created a chat client into a custom app and using the rest api to communicate with RocketChat.

I am uploading files over the function "/api/v1/rooms.upload", but the return value of this call is only if it was successful or not. Is there any chance to get a message object with the attachment information’s? Because I want to show the uploaded file at the messages in my app.

At this time I create a message object in the frontend, but I think that is not the best way to show the uploaded file as a message.

Bye Andreas H.

### PR DESCRIPTION
**Pull request by knrt10**
Closes #13808 

[Docs PR](https://github.com/RocketChat/docs/pull/1148)

@sampaiodiego @tassoevan would you please review.

### CODE REVIEW

**Comment by knrt10** [comment]
@MarcosSpessatto hmm, my bad. Let me fix this

**Comment by knrt10** [comment]
@MarcosSpessatto updated the PR.

File: app/api/server/v1/rooms.js [hunk 1]
```diff
@@ -118,12 +120,16 @@ API.v1.addRoute('rooms.upload/:rid', { authRequired: true }, {
 			delete fields.description;
 
 			API.v1.success(Meteor.call('sendFileMessage', this.urlParams.rid, null, uploadedFile, fields));
+
+			fileData = uploadedFile;
 		});
 
-		return API.v1.success();
+		return API.v1.success({ file: Messages.getMessageByFileIdAndUsername(fileData._id, this.userId) });
```
Comment (MarcosSpessatto) [inline app/api/server/v1/rooms.js]: Can you change `file` to `message`?

**Comment by knrt10** [comment]
No problem, let me update tests

**Comment by knrt10** [comment]
@MarcosSpessatto  tests updated, also a small fix was made when `fieldname` was other than `file`. Previously it was not sending a resposne. Now working fine. Please review.

**Comment by MarcosSpessatto** [comment]
@knrt10 amazing. Thanks so much. :1st_place_medal:

**Comment by knrt10** [comment]
No problem, happy to contribute

**Pull request merged**